<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="OpenScreen">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Rockwell</family>
            <pointsize>26</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>MultiplayerGame++</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="join">
            <property name="text">
             <string>Join</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="host">
            <property name="text">
             <string>Host</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="credits">
            <property name="text">
             <string>Credits</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="CreditsScreen">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="title">
          <property name="font">
           <font>
            <family>Rockwell</family>
            <pointsize>26</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>About MultiplayerGame++</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="about">
          <property name="font">
           <font>
            <family>Rockwell</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Originally created in python as a mock NEA for Alevel Computer science. By Oliver Killane.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Return">
          <property name="font">
           <font>
            <family>Rockwell</family>
            <weight>75</weight>
            <bold>true</bold>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Return</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="HostStartScreen">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="title_3">
          <property name="font">
           <font>
            <family>Rockwell</family>
            <pointsize>26</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Select game type.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="0">
           <widget class="QLabel" name="maplabel">
            <property name="font">
             <font>
              <family>Rockwell</family>
             </font>
            </property>
            <property name="text">
             <string>Map</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QComboBox" name="gamemode"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="gamemodelabel">
            <property name="font">
             <font>
              <family>Rockwell</family>
             </font>
            </property>
            <property name="text">
             <string>Game Mode</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QComboBox" name="map"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="maxplayerslabel">
            <property name="font">
             <font>
              <family>Rockwell</family>
             </font>
            </property>
            <property name="text">
             <string>Max Players</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QLabel" name="maxplayersshown">
              <property name="font">
               <font>
                <family>Rockwell</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>5</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignHCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="maxplayers">
              <property name="minimum">
               <number>5</number>
              </property>
              <property name="maximum">
               <number>30</number>
              </property>
              <property name="pageStep">
               <number>10</number>
              </property>
              <property name="sliderPosition">
               <number>5</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBelow</enum>
              </property>
              <property name="tickInterval">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="startgame">
          <property name="font">
           <font>
            <family>Rockwell</family>
           </font>
          </property>
          <property name="text">
           <string>Start Game</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="return_2">
          <property name="text">
           <string>Return</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="HostScreen">
       <layout class="QVBoxLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="title_2">
            <property name="font">
             <font>
              <family>Rockwell</family>
              <pointsize>26</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Host</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="PlayerTable"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Host info</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Return_2">
            <property name="text">
             <string>Return</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="JoinScreen">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="Title">
          <property name="font">
           <font>
            <family>Rockwell</family>
            <pointsize>26</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Join a Game</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="EnterName">
          <property name="font">
           <font>
            <family>Rockwell</family>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>Enter Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SelectColour">
          <property name="font">
           <font>
            <family>Rockwell</family>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Select Colour</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="EnterIP">
          <property name="font">
           <font>
            <family>Rockwell</family>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>Enter IP Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="JoinServer">
          <property name="font">
           <font>
            <family>Rockwell</family>
           </font>
          </property>
          <property name="text">
           <string>Join Server</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Return_3">
          <property name="text">
           <string>Return</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ConnectingScreen">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="loadinglabel">
          <property name="font">
           <font>
            <family>Rockwell</family>
            <pointsize>26</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Loading...</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="CancelConnection">
          <property name="font">
           <font>
            <family>Rockwell</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="GameScreen">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ErrorScreen">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="ErrorMessage">
          <property name="text">
           <string>An Error has occured, unable to continue connection to server.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="MenuReturn">
          <property name="text">
           <string>RETURN TO MAIN MENU</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
